packages/create-app/templates/default-app/app-config.yaml.hbs

```diff
@@ -6,6 +6,10 @@ organization:
   name: My Company
 
 backend:
+  # Used for enabling authorization, secret is shared by all backend plugins
+  # See authenticate-api-requests.md in the contrib docs for information on the format
```

Comment:
👍 on adding this bit. Gotta make sure it's in sync with wherever we end up with the docs

Reply:
Synced up in f5c43945eb